### PR TITLE
codeowners update

### PR DESCRIPTION
**Pull request by MichielBrys**
**Closes #<ISSUE_NUMBER>**

## PR Checklist

- [x] Created a feature/dev branch in your fork (vs. submitting directly from a commit on main)
- [x] Based off latest main branch of toolkit
- [x] PR doesn't include merge commits (always rebase on top of our main, if needed)
- [x] New integration
  - [x] Docs are written
  - [x] Added description of major feature to project description for NuGet package (4000 total character limit, so don't push entire description over that)
- [x] Tests for the changes have been added (for bug fixes / features) (if applicable)
- [x] Contains **NO** breaking changes
- [x] Every new API (including internal ones) has full XML docs
- [x] Code follows all style conventions

### CODE REVIEW

**Review by aaronpowell — APPROVED** [review]

**Comment by aaronpowell** [comment]
@michielb I've invited you to the contributor team which will resolve the error on the CODEOWNERS file

File: CODEOWNERS [hunk 1]
```diff
@@ -50,3 +50,11 @@
...
 /src/CommunityToolkit.Aspire.EventStore/_ @fredimachado
 /tests/CommunityToolkit.Aspire.EventStore.Tests/_ @fredimachado
+
+
+# CommunityToolkit.Aspire.MassTransit.RabbitMQ
+
+/examples/masstransit-rabbitmq/_ @michielb
+/src/CommunityToolkit.Aspire.MassTransit.RabbitMQ/_ @michielb
+/tests/CommunityToolkit.Aspire.MassTransit.RabbitMQ.Tests/_ @michielb
+
```
Comment (aaronpowell) [inline CODEOWNERS:60]:
```suggestion
/examples/masstransit-rabbitmq/ @michielb
/src/CommunityToolkit.Aspire.MassTransit.RabbitMQ/ @michielb
/tests/CommunityToolkit.Aspire.MassTransit.RabbitMQ.Tests/ @michielb

```

**Review by aaronpowell — CHANGES_REQUESTED** [review]

**Review by aaronpowell — APPROVED** [review]

**Comment by aaronpowell** [comment]
Bummer, I can't fix the merge issue on mobile

**Comment by Alirexaa** [comment]
> Bummer, I can't fix the merge issue on mobile

I will do.

**Comment by MichielBrys** [comment]
Oh I made a mistake, its supposed to be @MichielBrys

File: CODEOWNERS [hunk 2]
```diff
@@ -50,3 +50,12 @@
 /tests/CommunityToolkit.Aspire.Hosting.EventStore.Tests/ @fredimachado
 /src/CommunityToolkit.Aspire.EventStore/ @fredimachado
 /tests/CommunityToolkit.Aspire.EventStore.Tests/ @fredimachado
+
+# CommunityToolkit.Aspire.MassTransit.RabbitMQ
+
+/examples/masstransit-rabbitmq/ @michielb
+/src/CommunityToolkit.Aspire.MassTransit.RabbitMQ/ @michielb
+/tests/CommunityToolkit.Aspire.MassTransit.RabbitMQ.Tests/ @michielb
```
Comment (aaronpowell) [inline CODEOWNERS:58]:
```suggestion
/examples/masstransit-rabbitmq/ @MichielBrys
/src/CommunityToolkit.Aspire.MassTransit.RabbitMQ/ @MichielBrys
/tests/CommunityToolkit.Aspire.MassTransit.RabbitMQ.Tests/ @MichielBrys
```

**Comment by aaronpowell** [comment]
> Oh I made a mistake, its supposed to be @MichielBrys

I've proposed a suggested change, is that right?

**Review by aaronpowell — CHANGES_REQUESTED** [review]
Verify the right owner is set

**Comment by MichielBrys** [comment]
Yes thats correct, sorry for the delay. Thanks :)

**Review by aaronpowell — APPROVED** [review]

**Comment by aaronpowell** [comment]
> Yes thats correct, sorry for the delay. Thanks :)

Make sure you accept the invite that was sent then it's good to merge (otherwise the file is marked as invalid since you're not a member)

**Comment by MichielBrys** [comment]
Thanks for the invite Aaron, I wasn't available the last few days and it seems the invitation has been revoked or expired. Could you please send another one? I'll accept that one first thing tomorrow. My apologies :)

**Comment by aaronpowell** [comment]
> Thanks for the invite Aaron, I wasn't available the last few days and it seems the invitation has been revoked or expired. Could you please send another one? I'll accept that one first thing tomorrow. My apologies :)

done

**Comment by MichielBrys** [comment]
invite accepted

**Pull request merged**